File: backend/src/main/java/by/fpmibsu/findafriend/controller/commands/users/UpdateUserCommand.java
package by.fpmibsu.findafriend.controller.commands.users;

import by.fpmibsu.findafriend.application.mediatr.Request;

public class UpdateUserCommand extends Request<Boolean> {
    public int userId;
    public String name;
    public String surname;
    public String login;
    public String password;
    public String phoneNumber;
    public String email;
    public String providedPassword;

    public UpdateUserCommand() {
    }
}

File: BackendNET/Backend/Commands/Users/UpdateUser/UpdateUserCommand.cs
using MediatR;

namespace Backend.Commands.Users.UpdateUser;

public class UpdateUserCommand : IRequest<bool>
{
    public int UserId { get; set; }
    public string? Name { get; set; }
    public string? Surname { get; set; }
    public string? Login { get; set; }
    public string? Password { get; set; }
    public string? Email { get; set; }
    public string? PhoneNumber { get; set; }
}